cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -134,8 +134,14 @@ def count_records_rollups_table(db, provider):
     ems = db['ext_management_systems']
     provider_id = db.session.query(ems).filter(ems.name == provider.name).first().id
     rollups = db['metric_rollups']
-
-    count = db.session.query(rollups).filter(rollups.parent_ems_id == provider_id,
+    vms = db['vms']
+
+    if version.current_version() > 5.6:
```

Comment:
**Optional**
Comment (in code) as to why 5.7 has additional timestamp and cpu filters